UM/Controller.py

```diff
@@ -194,7 +195,18 @@ def removeInputDevice(self, name: str) -> None:
             self._input_devices[name].event.disconnect(self.event)
             del self._input_devices[name]
 
-    ##  Request tool by name. Returns None if no view is found.
+    ##  Request the current fallbacl tool.
+    #   \return Id of the fallback tool
+    def getFallbackTool(self) -> str:
+        return self._fallback_tool
+
+    ##  Set the current active tool. The tool must be set by name.
+    #   \param tool The tools name which shall be used as fallback
+    def setFallbackTool(self, tool: str):
```

Comment:
```suggestion
    def setFallbackTool(self, tool: str) -> None:
```
It needs to have full typing (so also indicate that nothing is returned)